src/dnstable_lookup.c

```diff
@@ -86,29 +94,41 @@ main(int argc, char **argv)
 	if (argc < 3)
 		usage();
 
-	if (strcmp(argv[1], "rrset") == 0) {
+	switch (getopt(argc, argv, "j")) {
```

Comment:
I am not comfortable with the readability/fragility of this code.  Would any of the enhanced getopt libraries improve this, or is it not worth it?